jclklib/client/jclk_init.cpp

```diff
@@ -169,9 +169,27 @@ bool JClkLibClientApi::jcl_disconnect()
     return retVal;
 }
 
+long long timespec_delta(const timespec& last_notification_time, const timespec& current_time) {
+    long long delta_in_sec = current_time.tv_sec - last_notification_time.tv_sec;
+    long long delta_in_nsec = current_time.tv_nsec - last_notification_time.tv_nsec;
+    long long delta_in_ms = delta_in_sec * 1000LL + delta_in_nsec / 1000000LL;
```

Comment:
suggest to use int64_t, instead of long long, to make sure same length is used in both 32bits and 64bits systems.